src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/components/call_outs/__tests__/__snapshots__/call_outs.test.js.snap

```diff
@@ -9,17 +9,26 @@ exports[`CallOuts should render normally 1`] = `
     title="Deprecation languages in use"
   >
     <p>
-      The following deprecated languages are in use: 
-      php
-      . Support for these languages will be removed in the next major version of Kibana and Elasticsearch. Convert you scripted fields to 
-      <EuiLink
-        color="primary"
-        href="http://www.elastic.co/painlessDocs"
-        type="button"
-      >
-        Painless
-      </EuiLink>
-       to avoid any problems.
+      <FormattedMessage
+        defaultMessage="The following deprecated languages are in use: {deprecatedLangsInUse}. Support for these languages will be removed in the next major version of Kibana and Elasticsearch. Convert you scripted fields to {link} to avoid any problems."
+        id="kbn.management.editIndexPattern.scripted.deprecationLangLabel.deprecationLangDetail"
+        values={
+          Object {
+            "deprecatedLangsInUse": "php",
+            "link": <EuiLink
+              color="primary"
+              href="http://www.elastic.co/painlessDocs"
+              type="button"
+            >
+              <FormattedMessage
+                defaultMessage="Painless"
```

Comment:
Since this is a proper name (the name of the scripting language) I would agree that it probably should not be translated.  I assume we don't translate "Elastic" for example.